File: src/com/mega/android/UserCredentials.java
package com.mega.android;

public class UserCredentials {
	private String email;
	private String publicKey;
	private String privateKey;
	
	public UserCredentials(String email, String privateKey, String publicKey) {
		this.email = email;
		this.privateKey = privateKey;
		this.publicKey = publicKey;
	}
	
	public String getEmail() {
		return email;
	}
	
	public String getPrivateKey() {
		return privateKey;
	}
	
	public String getPublicKey() {
		return publicKey;
	}
}

File: app/src/main/java/mega/privacy/android/app/domain/entity/user/UserCredentials.kt
package mega.privacy.android.app.domain.entity.user

/**
 * User credentials
 *
 * @property email
 * @property session
 * @property firstName
 * @property lastName
 * @property myHandle
 */
data class UserCredentials(
    val email: String,
    val session: String,
    val firstName: String,
    val lastName: String,
    val myHandle: String,
)
